SystemServices/SystemServices.cpp

```diff
@@ -3874,12 +3877,17 @@ namespace WPEFramework {
             int seconds = 600; /* 10 Minutes to Reboot */
 
             LOGINFO("len = %lud\n", len);
+            int state = 0;
```

Comment:
please remove changes in SystemServices. these changes make no sense

Reply:
reverted